packages/api-bindings/src/apollo/cache/type-policies/createProfileStatsTypePolicy.ts

```diff
@@ -26,6 +31,14 @@ export function createProfileStatsTypePolicy(): StrictTypedTypePolicies['Profile
           return existing ?? 0;
         },
       },
+      followers: {
+        read(existing: number | undefined, { readField }: FieldFunctionOptions) {
+          const profileId = readField('id') as ProfileId;
+          const pendingFollowTxCount = countPendingFollowFor(profileId);
+          const pendingUnfollowTxCount = countPendingUnfollowFor(profileId);
+          return (existing ?? 0) + pendingFollowTxCount - pendingUnfollowTxCount;
```

Comment:
I was about to say this cannot really happen, as in having pending follow and unfollow tx for the same profile id... but I believe it might happen with self-funded transactions in some edge cases circumstances. No harm in this though.